zuul.d/jobs.yaml

```diff
@@ -56,3 +56,24 @@
     name: edpm-ansible-crc-podified-edpm-baremetal
     parent: cifmw-crc-podified-edpm-baremetal
     irrelevant-files: *molecule_irrelevant_files
+
+# non-containerized role jobs
+- job:
+    name: edpm-ansible-edpm_libvirt
+    description: |
+      This job tests installing libvirt via the edpm_libvirt role.
+    parent: cifmw-molecule-base
```

Comment:
i think this need required-project for the ci framework

https://github.com/openstack-k8s-operators/ci-framework/blob/d778a092ab6f6c0031f9edce6804573188a2af26/ci/playbooks/molecule-prepare.yml#L22-L25